APIClient/Program.cs

```diff
@@ -7,18 +7,32 @@
 
 namespace ProductStoreClient
```

Comment:
ProductStoreClient?

Reply:
idk what is it, i  never seen before, i think was in the http client from the start

Reply:
change it